cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -140,13 +141,27 @@ void AnimatedImageTracker::Reset() {
 }
 
 void AnimatedImageTracker::OnRenderTreeRasterized() {
+  bool is_main_webmodule = name_ == "MainWebModule";
   count_animated_images_active = playing_urls_.size();
+  if (is_main_webmodule) {
+    UmaHistogramExactLinear("AnimatedImage.MainWebModule.Active",
```

Comment:
This is likely the wrong histogram to use here and throughout. Exact linear creates one bucket for every whole integer value from 1 to the max, in this case UINT_MAX. This would create billions of buckets. In general, Chrome advocates for _at most_ 100 buckets... and even less (e.g., 50) is a better choice. More on this here: https://chromium.googlesource.com/chromium/src/tools/+/HEAD/metrics/histograms/README.md#count-histograms and https://source.chromium.org/chromium/chromium/src/+/main:base/metrics/histogram_functions.h;l=35-46;drc=3f166bc0f254a0e0b9372955366898b5f2ba6909. 

Do you have a general idea what the min and max may be here? As a rule of thumb, only 1% of samples should ever have a chance of exceeding your maximum and making it into the overflow bucket. 

For example, if you're fairly sure these would stay under 100, you'd use `UmaHistogramCounts100`.

Reply:
> This is likely the wrong histogram to use here

I know :) I put this commit in more as a reminder to myself because the CVal work i did just above. In all likelihood we are mostly going to be interested in

- Number of simultaneously decoded images, that's going to be mostly 0 or 1 with range up to maybe 10, 20 with a very long tail
- Number of decoded frames over some interval. That could vary a lot, but probably in the range of 0 to few hundred
- Underruns as a percentage, overruns as a percentage of dropped frames.

Reply:
> Number of simultaneously decoded images, that's going to be mostly 0 or 1 with range up to maybe 10, 20 with a very long tail

K, sounds like some count histogram would be good here. UmaHistogramCounts100 actually might work nicely.

> Number of decoded frames over some interval. That could vary a lot, but probably in the range of 0 to few hundred

K Something like this then: `base::UmaHistogramCustomCounts("Cobalt...DecodedFrames", some_value, 0, 300, 50);`

> Underruns as a percentage, overruns as a percentage of decoded frames.

See percentage histograms: [UmaHistogramPercentage](https://source.chromium.org/chromium/chromium/src/+/main:base/metrics/histogram_functions.h;bpv=1;bpt=1;l=143?q=UmaHistogramCounts100&ss=chromium%2Fchromium%2Fsrc&gsn=UmaHistogramPercentage&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dbase%2Fmetrics%2Fhistogram_functions.h%23mJk845sbVYwTaRKiRESNYws58AvpcdOV7CfeLTCQO30)